clients/src/main/java/org/apache/kafka/common/requests/FetchMetadata.java

```diff
@@ -114,9 +114,21 @@ public boolean equals(Object o) {
     }
 
     /**
-     * Return the metadata for the next error response.
+     * Return the metadata for the next request. The metadata is set to indicate that the client wants to close the
+     * existing session.
      */
     public FetchMetadata nextCloseExisting() {
+        return new FetchMetadata(sessionId, FINAL_EPOCH);
```

Comment:
Yep, you can also check how the server code handles the $ID FINAL_EPOCH at https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/server/FetchSession.scala#L785

Could we make a quick change to correct the KIP for better documentation? Or do we have to go through community vote for it?

Reply:
Added the last row. We don't need another poll because we didn't change anything. We just added missing part in the KIP.

<img width="1495" alt="image" src="https://user-images.githubusercontent.com/43372967/189527055-936c4f46-18be-4b81-9874-a15ccb836196.png">